src/matter/cluster/FlowMeasurementCluster.ts

```diff
@@ -19,13 +19,13 @@ export const FlowMeasurementCluster = Cluster({
 
     /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.5.4 */
     attributes: {
-       /** MeasuredValue represents the flow in m3 /h as follows: MeasuredValue = 10 x Flow */
```

Comment:
@JimBuzbee nnooooooooo :-)) in this case where the comment comes directly before a codeline and is describing this codeline the /** .. */ for is the correct one!!

Reply:
:-) JSDoc - Add one more thing that I am a novice at! 

Correction pushed